include/albatross/src/core/model.hpp

```diff
@@ -59,8 +59,8 @@ template <typename ModelType> class ModelBase : public ParameterHandlingMixin {
                 !has_possible_fit<ModelType, FeatureType>::value &&
                     !has_valid_fit<ModelType, FeatureType>::value,
                 int>::type = 0>
-  void _fit(const std::vector<FeatureType> &features,
```

Comment:
the other option is (I think) leaving out the variable name,
```
void _fit(const std::vector<FeatureType> &,
```

Reply:
Yes -- `clang-tidy` complains about this, however.

Reply:
Ah weird, anyway this works!